driver-sync/src/test/functional/com/mongodb/client/unified/UnifiedTestSkips.java

```diff
@@ -17,26 +17,279 @@
+                        + "closing the cursor is something that happens under "
+                        + "user control")
+                .test("load-balancers", "cursors are correctly pinned to connections for load-balanced clusters", "pinned connections are not returned after an network error during getMore")
+                .test("load-balancers", "cursors are correctly pinned to connections for load-balanced clusters", "pinned connections are not returned to the pool after a non-network error on getMore");
+        def.skipNoncompliantReactive("Reactive streams driver can't implement "
+                        + "this test because there is no way to tell that a "
+                        + "change stream cursor that has not yet received any "
+                        + "results has even initiated the change stream")
+                .test("load-balancers", "cursors are correctly pinned to connections for load-balanced clusters", "change streams pin to a connection");
+
+        // crud
+
+        def.skipDeprecated("Deprecated count method removed, cf https://github.com/mongodb/mongo-java-driver/pull/1328#discussion_r1513641410")
+                .test("crud", "count-empty", "Deprecated count with empty collection")
+                .test("crud", "count-collation", "Deprecated count with collation")
+                .test("crud", "count", "Deprecated count without a filter")
+                .test("crud", "count", "Deprecated count with a filter")
+                .test("crud", "count", "Deprecated count with skip and limit");
+        def.skipTodo("See downstream changes comment on https://jira.mongodb.org/browse/JAVA-4275")
+                .test("crud", "findOneAndReplace-hint-unacknowledged", "Unacknowledged findOneAndReplace with hint string on 4.4+ server")
+                .test("crud", "findOneAndReplace-hint-unacknowledged", "Unacknowledged findOneAndReplace with hint document on 4.4+ server")
+                .test("crud", "findOneAndUpdate-hint-unacknowledged", "Unacknowledged findOneAndUpdate with hint string on 4.4+ server")
+                .test("crud", "findOneAndUpdate-hint-unacknowledged", "Unacknowledged findOneAndUpdate with hint document on 4.4+ server")
+                .test("crud", "findOneAndDelete-hint-unacknowledged", "Unacknowledged findOneAndDelete with hint string on 4.4+ server")
+                .test("crud", "findOneAndDelete-hint-unacknowledged", "Unacknowledged findOneAndDelete with hint document on 4.4+ server");
+        def.skipJira("https://jira.mongodb.org/browse/JAVA-5481") // TODO this is completed?
+                .onlyWhen(() -> isDiscoverableReplicaSet() && serverVersionAtLeast(8, 0))
+                .test("crud", "aggregate-write-readPreference", "Aggregate with $out includes read preference for 5.0+ server")
+                .test("crud", "db-aggregate-write-readPreference", "Database-level aggregate with $out includes read preference for 5.0+ server");
```

Comment:
This https://jira.mongodb.org/browse/JAVA-5481 is closed - possibly safe to remove these tests?